packages/fields/src/types/Relationship/Implementation.js

```diff
@@ -252,42 +249,44 @@ export class Relationship extends Implementation {
     // mutation createPost() {
     //   author: { connect: { id: 'abc123' } }
     // }
+    const operations = [];
     if (this.many) {
+      if (refList.access[schemaName].create) {
+        operations.push(`# Provide data to create a set of new ${refList.key}. Will also connect.
+          create: [${refList.gqlNames.createInputName}]`);
+      }
```

Comment:
A `create: ` operation during a relationship update is really just a "create + connect". If we can't `create` the target list (`refList.access[schemaName].create === false`) then we want to remove this as an option, but leave regular `connect` operations.